db/config.ts

```diff
@@ -12,6 +12,7 @@ const Openings = defineTable({
     companyname: column.text(),
     companyurl: column.text(),
     companycategory: column.text({ optional: true }),
+    created_at: column.date(),
```

Comment:
any reason why this isn't  `created_at: column.date({ default: NOW }),`?

Reply:
I left it blank because I didn't know if it would be supplied with the source data.

Reply:
@Barabazs & @vorcigernix tell me once it will be merged - I can actively test it & ask contributors to update the DB with actual vacancies.

Reply:
@vorcigernix any updates here?

Reply:
> any reason why this isn't created_at: column.date({ default: NOW }),?

fixed